scripts/ci/libraries/_docker_engine_resources.sh

```diff
@@ -22,27 +22,25 @@ function docker_engine_resources::print_overall_stats() {
     echo "Overall resource statistics"
     echo
     docker stats --all --no-stream --no-trunc
-    docker run --rm --entrypoint /bin/bash "${AIRFLOW_CI_IMAGE}" -c "free -h"
-    df --human || true
+    docker run --rm --entrypoint /bin/bash "debian:buster-slim" -c "cat /proc/meminfo"
+    df -H || true
```

Comment:
Is this intentional? `df --human` is 1024-based (same as `df -h`), but `df -H` is 100-based (same as `df --si`).

Otherwise lgtm.

Reply:
Right. I had to change to shorter version (MacOS does not have `--human` switch - which is funny BTW).